datasets/ONS-UK-trade-in-services-by-business-characteristics/info.json

```diff
@@ -18,7 +18,1134 @@
         "stage": [
             "To Do"
         ],
-        "main_issue": 89
+        "main_issue": 89,
+        "transformStage": [
```

Comment:
Can you remove the `transformStage` stuff? It makes it very hard to check if you've added any `columns` to this file. 

Have you started considering how your columns will link up to other external dimensions? Any thoughts on which measure and units you're going to use?